sympy/physics/control/lti.py

```diff
@@ -0,0 +1,553 @@
+
+        TransferFunction
+            Resultant Transfer Function after dividing (with '/' operator) self by other.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(s + 1, s - 8, s)
+        >>> tf2 = TransferFunction(p + s**2, p - s, s)
+        >>> tf1/tf2
+        TransferFunction((p - s)*(s + 1), (p + s**2)*(s - 8), s)
+        >>> tf2/(-3)
+        TransferFunction(p + s**2, -3*p + 3*s, s)
+        >>> tf1/(a**2)
+        TransferFunction(s + 1, a**2*(s - 8), s)
+
+        """
+        other = _sympify(other)
+        if other.is_number:
+            if other == 0:
+                raise ValueError("TransferFunction cannot be divided by zero.")
+            return TransferFunction(self.num, self.den*other, self.var)
+
+        elif isinstance(other, TransferFunction):
+            if not self.var == other.var:
+                raise ValueError("Both the Transfer Functions should be anchored "
+                    "with the same variable.")
+            p = self.num * other.den
+            q = self.den * other.num
+            return TransferFunction(p, q, self.var)
+
+        elif isinstance(other, Expr) and other.has(Symbol):
+            return TransferFunction(self.num, self.den*other, self.var)
+        else:
+            raise ValueError("TransferFunction cannot be divided by {}.".
+                format(type(other)))
+
+    __truediv__ = __div__
+
+    def __rtruediv__(self, other):
+        return _sympify(other) * self**-1
+
+    __rdiv__ = __rtruediv__
+
+    def __pow__(self, p):
+        """Finds the p-th power of the Transfer Function.
+
+        Parameters
+        ==========
+
+        p : int
+            Power to be applied on the Transfer Function.
+
+        Returns
+        =======
+
+        TransferFunction
+            Returns the p-th power of the current Transfer Function.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(s + a, s - a, s)
+        >>> tf1**3
+        TransferFunction((a + s)**3, (-a + s)**3, s)
+        >>> tf2 = TransferFunction(s, s**2 + p, s)
+        >>> tf2**0
+        TransferFunction(1, 1, s)
+        >>> tf3 = TransferFunction(p + 4, p - 3, p)
+        >>> tf3**-1
+        TransferFunction(p - 3, p + 4, p)
+
+        """
+        p = sympify(p)
+        if not isinstance(p, Integer):
+            raise ValueError("Exponent must be an Integer.")
+        if p == 0:
+            return TransferFunction(1, 1, self.var)
+        elif p > 0:
+            num_, den_ = self.num**p, self.den**p
+        else:
+            p = abs(p)
+            num_, den_ = self.den**p, self.num**p
+
+        return TransferFunction(num_, den_, self.var)
+
+    def __neg__(self):
+        """
+        Negates a Transfer Function.
```

Comment:
You started the docstring from a newline here while it was on the same line in the previous instances. Although both of them can be used, and I don't think any one of them is preferred. But, whichever one you decide to use should be used consistently. 
Don't mix both the styles through the docstrings. Stick to one.